### ISSUE TITLE
Simplifying `String.fromList` produces invalid code

### ISSUE THREAD

**Issue by tfausak** [jfmengels/elm-review-simplify#85]
When running `elm-review --fix` with code like this:

``` elm
String.fromList [ f x ]
```

The result is invalid:

``` elm
String.fromChar f x
```

I think it should be:

``` elm
String.fromChar (f x)
```

**Comment by tfausak** [comment]
Thanks for the quick fix @lue-bird! And thanks for the great tool @jfmengels!

**Comment by jfmengels** [comment]
Released in 2.0.27. Thanks @tfausak for reporting and @lue-bird for fixing this so quickly :heart:
